docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202304.0/security-release-notes-202304.0.md

```diff
@@ -0,0 +1,475 @@
+---
+title: Security release notes 202304.0
+description: Security release notes for the Spryker Product release 202304.0
+last_updated: Apr 21, 2023
+template: concept-topic-template
+---
+
+The following information pertains to security-related issues that have been recently resolved. All issues are listed by description and affected modules.
+
+If you need any additional support with this content, [contact our support](mailto:support@spryker.com). If you found a new security vulnerability, inform us through [security@spryker.com](mailto:security@spryker.com).
+
+
+## SQL injection in the Propel module
+
+An attacker can inject malicious code in functionalities of the Spryker application that are using the PropelFilterCriteria function. The injected code is then executed in the underlying database. The SQL injection vulnerability affects the Propel module that handles the interactions of the application with the database.
+
+### Affected modules
+
+  `spryker/propel`: 1.0.0-3.37.0
+
+### How to get the fix
+
+Update the affected module `spryker/propel` to version 3.38.0.
+```bash
+composer require spryker/propel:"~3.38.0"
+composer show spryker/propel # Verify the version
+```
+
+## Stored Cross-Site Scripting (XSS) in Marketplace
+
+An attacker can inject malicious payloads into the merchant profile page in the Merchant portal that's executed in the Back Office. This vulnerability lets an attacker execute code in the context of the Back Office user.
+
+### Affected modules
+
+* `spryker/merchant-profile-merchant-portal-gui`: 1.0.0-2.1.0
+* `spryker/product-merchant-portal-gui`: 1.0.0-3.3.1
+
+### How to get the fix
+
+For the module `spryker/product-merchant-portal-gui`, depending on the version you are using, follow these steps:
+
+* If your version of `spryker/product-merchant-portal-gui` is earlier than 2.0.0, update to version 1.7.1:
+```bash
+composer require spryker/product-merchant-portal-gui:"~1.7.1" spryker/gui:"~3.48.0" spryker/util-sanitize-xss:"~1.1.0"
+composer show spryker/product-merchant-portal-gui # Verify the version
+```
+
+* If your version of `spryker/product-merchant-portal-gui` is 2.0.0 up to and including 3.0.0, update to version 2.15.1:
```

Comment:
"2.0.0 up to" - does it mean later than 2.0.0?

Reply:
That's the phrase devs want to use to make sure it's obvious that the version range includes 2.0.0 and 3.0.0.

Reply:
maybe then "2.0.0 and later, including 3.0.0"? But it is still up to you to decide on the wording here